dist/textbox/ds6/textbox.css

```diff
@@ -64,6 +64,7 @@ input.textbox__control--floated,
 textarea.textbox__control--floated {
```

Comment:
Are we allowing floated labels for textarea? I didn't think so.

Reply:
good catch. Moved it.